### ISSUE TITLE
 Help with the interview to the creator of OpenAI

### ISSUE THREAD

**Issue by Root-FTW** [abdeladim-s/subsai#17]
I have been using your project for several days, and it has been working more than perfect, but today I got the idea to get the .sdr file from Lex Fridman's interview to OpenAI's creator.

[](https://www.youtube.com/watch?v=L_Guz73e6fw)

as you can see the video lasts more than 2 hours, and when I try to process it with your tool I get this error:
![image](https://user-images.githubusercontent.com/108767514/229337167-28930486-cfee-498a-a198-067acc0fdb38.png)

Is there any way to get the .srt file of such a long video using your tool?

thank you for this beautiful project, I hope you can help me.

**Comment by abdeladim-s** [comment]
@Root-FTW, Yes for such a long video, it is probably better to use a Python script. 

But I don't know if you read the README or not! but I have made an example script to transcribe long media files using a `VAD`, it will transcribe just the fragments when the voice is detected, and it will also save the process so you can stop/resume whenever you want.

> * [VAD example](https://github.com/abdeladim-s/subsai/blob/main/examples/subsai_vad.ipynb): process long audio files using [silero-vad](https://github.com/snakers4/silero-vad). <a target="_blank" href="https://colab.research.google.com/github/abdeladim-s/subsai/blob/main/examples/subsai_vad.ipynb">
  <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
</a>

Hope this will help!
